### ISSUE TITLE
Need assistance in using -include and -include_lib

### ISSUE THREAD

**Issue by SenseiC** [erlang/erlide_eclipse#79]
Trying to do Erlang development with the RabbitMQ Erlang Client.  To do that requires including header files:

-include("amqp_client.hrl").

(http://www.rabbitmq.com/erlang-client-user-guide.html#headers) and, of course, THAT file contains some include_lib statements:

-include_lib("rabbit_common/include/rabbit.hrl").
-include_lib("rabbit_common/include/rabbit_framing.hrl").

Now I have tried adding a relative path to the location of the amqp_client.hrl file via Windows > Preferences > Erlang > Compiler in the "Additional include dirs:" field, but the compiler still complains it can'f find the file.

I tried right-click on my Erlang project, clicking on Properties and then Erlang, but when I click on the include field's New... button, it only presents me with folders specifically in that Erlang Project.  I had tried adding references of this project to a project I had downloaded from the RabbitMQ's Mercurial repository (built the code from a terminal window and then refreshed the project in Eclipse), but it wouldn't appear in the list presented by the New... button.

Basically I have to be missing SOMETHING, but every "solution" I have found related to ERLIDE and -include/-include_lib hasn't helped this "blind man" exclaim, "I see!".

Now in return if I CAN get this to work, I will GLADLY provide images, etc. for review/approval as an addition to the Wiki.   8^D

Jay

PS: I would gladly discuss this off-line or in some forum, if you'll provide me with where to go for that (say the Eclipse forums as an example).
